shortlister/controller.py

```diff
@@ -66,12 +68,31 @@ def edit_criteria_select(self, k=None):
         
     def edit_score_confirm(self,k=None):
         """Confirm changes to score"""
-        
         self.view.view_update(self.current_criterion.name,self.current_criterion.scores[int(k)])
-        update_applicant_score(self.current_applicant,self.current_criterion,int(k))
-        self.view.view_applicant_details(self.current_applicant)
+        update_applicant_score(self.applicant(self.applicant_index),self.current_criterion,int(k))
+        self.view.view_applicant_details(self.applicant(self.applicant_index))
         self.options = self.options_applicant_detail
 
+    def switch_prev_applicant(self,k=None):
+        """shows details of the previous applicant in the shortlist"""
+        if self.applicant_index == 0:
+            pass
+        else:
+            self.applicant_index -=1
+            self.view.view_applicant_details(self.applicant(self.applicant_index))
```

Comment:
You can simplify this to simple ignore the case of when you're at the first applicant:

```suggestion
        """shows details of the previous applicant in the shortlist"""
        if self.applicant_index > 0:
            self.applicant_index -=1
            self.view.view_applicant_details(self.applicant(self.applicant_index))
```